lib/SILOptimizer/IPO/GlobalOpt.cpp

```diff
@@ -636,6 +636,10 @@ static SILValue convertLoadSequence(SILValue oldSequence,
     return newValue;
   }
 
+  if (auto *beginAccess = dyn_cast<BeginAccessInst>(oldSequence)) {
+    return convertLoadSequence(beginAccess->getOperand(), newRootValue, B);
```

Comment:
Dropping the access marker?

Reply:
My thinking was that it was no longer needed because `newRootValue` should be a loaded value. That may not be correct, though.

Reply:
You're right. Looking a little deeper, the load is replaced by a getter which will have its own access markers.